sys/compat/linprocfs/linprocfs.c

```diff
@@ -126,6 +126,9 @@
 #define P2K(x) ((x) << (PAGE_SHIFT - 10))		/* pages to kbytes */
 #define TV2J(x)	((x)->tv_sec * 100UL + (x)->tv_usec / 10000)
 
+/* Value defined in sys/kern/sysv_sem.c */
```

Comment:
Where?

Reply:
> Where?

Sorry. It's `sys/kern/sysv_shm.c`:

https://github.com/freebsd/freebsd-src/blob/main/sys/kern/sysv_shm.c#L114

Fixed.